api/controller/blog.js

```diff
@@ -6,7 +8,46 @@ exports.getBlog = function(req,res) {
 	Blog.findById(req.params.blog_id,function(err,blog) {
 		if (err)
 			res.send(err);
-		
+
 		res.json(blog);
 	});
-};
\ No newline at end of file
+};
+
+exports.deleteBlog = function(req, res) {
+	Blog.findById(req.params.blog_id, function(err, blog) {
+		if (err) {
+			res.status(500).json(err);
+			return;
+		}
+
+		if (blog != undefined) {
+			apiRoute.AuthChecker(req, res, function() {
```

Comment:
There is  no need to call AuthChecker api. When user sign in , req.user will get set  automatically .